specs/credentialmanagement/index.src.html

```diff
@@ -821,9 +867,10 @@ <h4 id="interfaces-request-options">
 
     <pre>
       navigator.<a idl>credentials</a>.<a idl lt="get()">get</a>({
-        "<a idl>types</a>": [ "federated" ],
-        "<a idl>options</a>": {
-            "<a idl>providers</a>": [ "https://identity.example.com/" ]
+        "<a idl>filters</a>": {
+            "federated" : {
+                "<a idl>providers</a>": [ "https://identity.example.com/" ]
```

Comment:
I certainly prefer it but as you say it may get unwieldy if the list of global arguments grows. Couldn't they also go in the `options` object? If the y doesn't match a type in the list of types it's assumed to be a global option?